packages/edit-site/src/components/dataviews/add-filter.js

```diff
@@ -81,6 +81,8 @@ export default function AddFilter( { fields, view, onChangeView } ) {
 						{ filter.elements.map( ( element ) => (
 							<DropdownMenuItem
 								key={ element.value }
+								role="menuitemradio"
+								aria-checked={ false }
```

Comment:
This is a weird one, but I'd probably argue that these should have a role of `menuitem`. They are not values in and of themselves; instead, they are all actions that create a new filter with a particular value.

```suggestion
								role="menuitem"
```

Ideally we'd put this in front of actual users, but given the submenu isn't available once the filter has been created, it kind of ceases to matter what their selected state is, or whether they are mutually exclusive.

Reply:
(Looks like I was 30 seconds late with feedback 😩 Not a blocker, just something to think about.)

Reply:
Ugh, sorry, had this to "auto-merge" once test pass. Feedback is never late, though :) https://github.com/WordPress/gutenberg/pull/56714

Also incorporated this into the refactor to ariakit: https://github.com/WordPress/gutenberg/pull/55625